### PR TITLE
Add blog post regarding 2.0.0 GA release

### PR DESCRIPTION
**Pull request by cdrage**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

/kind documentation
[skip ci]

**What does does this PR do / why we need it**:

Adds a blog post which will go live on September 23rd regarding the
2.0.0 release.

**Which issue(s) this PR fixes**:

N/A

**PR acceptance criteria**:

- [X] Unit test

- [X] Integration test

- [X] Documentation

- [X] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: docs/blog/2020-09-24-odo-200-ga-release.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+---
+
+`2.0.0` of odo has been released! 
+
+# What's new
+
+#### Changes to the default deployment method
+
+[Devfile](ihttps://devfile.github.io/devfile/index.html) is a file format that is used as odo's new deployment engine. Starting from `2.0.0` onwards, Source-to-Image (S2I) is no longer the default deployment method. S2I is still supported and can now be accessed with the `--s2i` flag from the command-line.
```
Comment (maysunfaisal) [inline docs/blog/2020-09-24-odo-200-ga-release.md:32]: link is broken

File: docs/blog/2020-09-24-odo-200-ga-release.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  etcdoperator.v0.9.4           EtcdCluster, EtcdBackup, EtcdRestore
+
+$ odo service create etcdoperator.v0.9.4/EtcdCluster
+```
+
+#### `odo debug` is no longer in technical preview
+
+The `odo debug` command is no longer in technical preview.
+
+[Learn how to debug your computer via the CLI or VSCode](https://odo.dev/docs/debugging-using-devfile/).
```
Comment (maysunfaisal) [inline docs/blog/2020-09-24-odo-200-ga-release.md:96]: shouldnt it be `Learn how to debug your component* via the CLI or VSCode`?

File: docs/blog/2020-09-24-odo-200-ga-release.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
+---
+# Layout settings, do not change
+layout: default
+keywords:
+comments: false
+micro_nav: true
+
+# Blog post information
+title: Odo 2.0.0 General Availability Release!
+description: odo has reached 2.0.0 and were excited to show you what's new
```
Comment (maysunfaisal) [inline docs/blog/2020-09-24-odo-200-ga-release.md:10]: we're* typo

**Comment by maysunfaisal** [comment]
I am not sure but i would also add the odo slack channel `odo-users` info in here somwhere so that anyone reading can comment if they have questions or anything to say.

File: docs/blog/2020-09-24-odo-200-ga-release.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+
+`2.0.0` of odo has been released! 
+
+# What's new
+
+#### Changes to the default deployment method
+
+[Devfile](ihttps://devfile.github.io/devfile/index.html) is a file format that is used as odo's new deployment engine. Starting from `2.0.0` onwards, Source-to-Image (S2I) is no longer the default deployment method. S2I is still supported and can now be accessed with the `--s2i` flag from the command-line.
+
+Learn how to deploy your first devfile from our [Devfile documentation](https://odo.dev/docs/deploying-a-devfile-using-odo/).
```
Comment (adisky) [inline docs/blog/2020-09-24-odo-200-ga-release.md:34]: Learn how to deploy your first application using devfiles?

File: docs/blog/2020-09-24-odo-200-ga-release.md (hunk 1 above)
Comment (boczkowska) [inline docs/blog/2020-09-24-odo-200-ga-release.md:32]:
```suggestion
[Devfile](https://devfile.github.io/devfile/index.html) is a file format that is used as odo's new deployment engine. Starting from `2.0.0` onwards, Source-to-Image (S2I) is no longer the default deployment method. S2I is still supported and can now be accessed with the `--s2i` flag from the command-line.
```

**Pull request merged**